api.yaml

```diff
@@ -138,6 +140,46 @@ paths:
             schema:
               $ref: '#/components/schemas/link_accounts'
         required: true
+  /users/role:
+    post:
+      tags:
+        - admins
+        - noroles
+      summary: Modifies the role of the logged user.
+      operationId: setRole
+      description: |
+        Receives the new role to be added to the user
+      responses:
+        '200':
+          description: Role updated
+        '400':
+          description: 'invalid input, object invalid'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/set_role_body'
+        required: true
```

Comment:
This should be a PATCH /users endpoint. Patch accepts a partial user, that is, just the properties that you want to patch. Basically, those properties are merged with the current user

Reply:
done